scylladb/sphinx-scylladb-theme
# Add Scylla Summit Banner

Adds the Scylla Summit Banner at the top of every documentation page. If the user clicks the cross button, the banner should not appear anymore, even if the user refreshes the page. The banner will be rolled out to every documentation project as soon as we publish a new version of the theme. 

## How to test this PR

Download the changes and run ``make preview``. If you close the banner but still want to see the banner, open ``localhost:5500`` in a new tab with incognito mode.

**Destkop preview**
![image](https://user-images.githubusercontent.com/9107969/99880646-a25a7900-2c0c-11eb-9093-6764c7f3b495.png)

**Tablet preview**

![image](https://user-images.githubusercontent.com/9107969/99880760-43e1ca80-2c0d-11eb-99ad-3842e31ab748.png)

**Mobile preview**

![image](https://user-images.githubusercontent.com/9107969/99880968-b43d1b80-2c0e-11eb-9f4b-55c2849db522.png)
